index.html

```diff
@@ -5,63 +5,259 @@
+        <div id="about">
+            <div id="box1">
+                <h2>About <br> Myself</h2>
+                <div class="intro">
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me
+                </div>
+                <div class="connect">LET'S CONNECT</div>
+                <div class="link">
+                    <ul>
+                        <li>
+                            <a class="Github" href="">
+                                <img src="images/header/github.png" alt="">
+                            </a>
+                        </li>
+                        <li>
+                            <a class="LinkedIn" href="">
+                                <img src="images/header/linkedin.png" alt="">
+                            </a>
+                        </li>
+                        <li>
+                            <a class="Ok" href="">
+                                <img src="images/header/ok.png" alt="">
+                            </a>
+                        </li>
+                        <li>
+                            <a class="Twitter" href="">
+                                <img src="images/header/twiter.png" alt="">
+                            </a>
+                        </li>
+                        <li>
+                            <a class="M" href="">
+                                <img src="images/header/m.png" alt="">
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+               <button class="box">Get my resume</button>
+            </div>
+            <div id="box2">
+                <div class="lang">
+                    <h3>Languages</h3>
+                    <span class="material-symbols-outlined">
+                        expand_more
+                    </span>
+                </div>
+                <div class="languages">
+                    <div class="js">
+                        <img src="images/header/js.png" alt="js">
+                        <h3>JavaScript</h3>
+                    </div>
+                    <div class="html">
+                        <img src="images/header/html.png" alt="html">
+                        <h3>HTML</h3>
+                    </div>
+                    <div class="css">
+                        <img src="images/header/css.png" alt="css">
+                        <h3>CSS</h3>
+                    </div>
+                </div>
+                <div class="framework">
+                    <h1>Framework</h1>
+                    <span class="material-symbols-outlined">
+                        chevron_right
+                    </span>
+                </div>
+                <div class="line"></div>
+                <div class="skills">
+                    <h1>Skills</h1>
```

Comment:
- [ ] Kindly take a look at [this article](https://www.w3.org/WAI/test-evaluate/preliminary/#headings) and make sure that the heading structure is following the correct hierarchy. The `h1` tag should be the main heading of the page and the page should not have more than one `h1` tag.